ArduPlane/AP_ExternalControl_Plane.h

```diff
@@ -16,6 +16,12 @@ class AP_ExternalControl_Plane : public AP_ExternalControl {
         Sets the target global position for a loiter point.
     */
     bool set_global_position(const Location& loc) override WARN_IF_UNUSED;
+
+    /*
+        Sets only the target yaw rate
+        Yaw is in earth frame, NED [rad/s]
```

Comment:
i'd like a comment on what the intention is with coordinated turns. I would expected we would implement it with roll and coordinated

Reply:
"A configured yaw rate of  2* pi / 30 will result in a complete circle in 30 seconds."